Marlin/src/module/planner.h

```diff
@@ -477,7 +477,7 @@ class Planner {
       #endif
     #endif
 
-    #if HAS_CLASSIC_JERK
+    #if ENABLED(CLASSIC_JERK)
       // (mm/s^2) M205 XYZ(E) - The largest speed change requiring no acceleration.
       static TERN(HAS_LINEAR_E_JERK, xyz_pos_t, xyze_pos_t) max_jerk;
```

Comment:
I'm arguing against my own interests here as I have another dependent change that I'd want to send a pull request for.. but since you're touching this code anyway: Would be good to clean up all instances of HAS_LINEAR_E_JERK guarded by CLASSIC_JERK - there are a couple more below. HAS_LINEAR_E_JERK is always undefined when CLASSIC_JERK.